suite-native/device-manager/src/components/DeviceItem/DeviceItem.tsx

```diff
@@ -0,0 +1,37 @@
+import { useSelector } from 'react-redux';
+import { Pressable } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { HStack } from '@suite-native/atoms';
+import { DeviceRootState, selectDeviceById } from '@suite-common/wallet-core';
+import { Icon } from '@suite-common/icons';
+import { TrezorDevice } from '@suite-common/suite-types';
+
+import { DeviceItemContent } from './DeviceItemContent';
+
+type DeviceItemProps = {
+    id: TrezorDevice['id'];
+    onPress?: () => void;
```

Comment:
Why is `onPress` optional? You should always have a press function here if this component gets rendered